src/SBKolmogorov.cpp

```diff
@@ -367,19 +367,14 @@ namespace galsim {
         // We use a cubic spline for the interpolation, which has an error of O(h^4) max(f'''').
         // As with Sersic (since Kolmogorov is just a Sersic with n=0.6 in reverse), we use
         // max(f'''') ~= 10.  So:
-        // 10 h^4 <= kvalue_accuracy
-        // h = (kvalue_accuracy/10)^0.25
+        // 10 h^4 <= xvalue_accuracy
+        // h = (xvalue_accuracy/10)^0.25
         double dlogr = gsparams->table_spacing * sqrt(sqrt(gsparams->xvalue_accuracy / 10.));
         xdbg<<"dlogr = "<<dlogr<<std::endl;
 
-        // Along the way accumulate the flux integral to determine the radius
-        // that encloses (1-folding_threshold) of the flux.
-        double sum = 0.;
-        double thresh0 = 0.5 / (2.*M_PI*dlogr);
-        double thresh1 = (1.-gsparams->folding_threshold) / (2.*M_PI*dlogr);
-        double thresh2 = gsparams->shoot_accuracy / (2.*M_PI*dlogr);
-        xdbg<<"thresh values = "<<thresh0<<"  "<<thresh1<<"  "<<thresh2<<std::endl;
-        double R = 0., hlr = 0.;
+        // Continue until the missing flux is less than shoot_accuracy.
+        double thresh = gsparams->shoot_accuracy / (2.*M_PI*dlogr);
```

Comment:
Can you double check the factor of `dlogr` here?  Since the integral below is analytic instead of a Riemann sum, I think it may be incorrect.

Reply:
Thank you!  This was indeed wrong.  The thing that gets compared to thresh now is just the integral divided by 2pi.